cmd/gossamer/config.go

```diff
@@ -581,7 +581,11 @@ func setDotCoreConfig(ctx *cli.Context, tomlCfg ctoml.CoreConfig, cfg *dot.CoreC
 	cfg.BabeAuthority = tomlCfg.Roles == types.AuthorityRole
 	cfg.GrandpaAuthority = tomlCfg.Roles == types.AuthorityRole
 	cfg.GrandpaInterval = time.Second * time.Duration(tomlCfg.GrandpaInterval)
-	cfg.BABELead = ctx.GlobalBool(BABELeadFlag.Name)
+
+	cfg.BABELead = tomlCfg.BABELead
+	if ctx.IsSet(BABELeadFlag.Name) {
+		cfg.BABELead = ctx.GlobalBool(BABELeadFlag.Name)
+	}
```

Comment:
Flag should always take precedence over the config file right? If so, all good :+1:

Reply:
yes